gateway/utils/syncman/operations.go

```diff
@@ -169,6 +169,16 @@ func (s *Manager) setProject(ctx context.Context, project *config.Project) error
 	return s.store.SetProject(ctx, project)
 }
 
+// SetAdminConfig applies the set Admin config
+func (s *Manager) SetAdminConfig(ctx context.Context, req *config.ClusterConfig) error {
+	// Acquire a lock
+	// s.lock.Lock()
+	// defer s.lock.Unlock()
+	c := s.GetGlobalConfig()
+	c.Admin.ClusterConfig = req
```

Comment:
acquire lock & use c.projectconfig.Admin.ClusterConfig instead of getglobal config